cli/charm_demo.ts

```diff
@@ -58,8 +58,8 @@ async function main() {
   );
 
   // let's add the cell to the charmManager
-  await charmManager.add([cell]);
-  log(charmManager, "charmmanager after adding cell");
+  // await charmManager.add([cell]);
+  // log(charmManager, "charmmanager after adding cell");
```

Comment:
?

Reply:
this is half-way written code by @ellyxir - I asked her if I can just comment out - she said yep